posthog/asgi.py

```diff
@@ -53,4 +61,48 @@ async def inner(scope, receive, send):
     return inner
 
 
+async def _prep_local_db_for_prod_reads():
+    """
+    When connecting to the prod DB locally, we need foreign keys associated with the user we're impersonating to exist locally.
+
+    Rationale:
+    If we don't ensure this, then we can't use the local DB for any writes. An example is the AI Conversation model:
+    for a Conversation to be started, it needs to be persisted. We can't do this in the prod DB, as we're only reading it,
+    so we must persist the Conversation locally. For that local write to work, the `team_id` and `user_id` foreign keys
+    must resolve in the local DB _as well_ as in the prod one!
+    """
+    from posthog.models import Organization, Project, Team, User
+
+    user: User = await User.objects.aget(email=settings.DEBUG_LOG_IN_AS_EMAIL)
+    try:
+        await User.objects.using("default").acreate(email=settings.DEBUG_LOG_IN_AS_EMAIL, id=user.id)
+    except Exception:
+        pass
+    try:
+        await Organization.objects.using("default").acreate(
```

Comment:
Paranoid: Add explicit checks for `user.current_organization_id` and `user.current_organization_id`, as they can be null?